proto_generator/protogenerator.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2017 Google Inc.
```

Comment:
Is this just a move with no changes? I don't recall if there's much history but it may be worth doing this with git mv.

Reply:
It's a new file -- we have `generator` and it seemed cleaner to add a new binary for doing the proto generation since there are some different behaviours here (e.g., writing out to a dir vs. writing out files).